diagnostic.md

```diff
@@ -18,58 +21,59 @@ In what file does the above code go?
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+/running db:migrate runs the change or up method for all the migrations that haven't been run.
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+// a scaffold generates your model, views, controller, and writes to your routes.rb file.
+rails g controller things just generates the controller
 ```
 
 What is the job of the `serializer`?
 
 ```md
-// your response here
+// returns a hash representing the model
 ```
 
 What do we expect the command `Patient.all` to return?
 
 ```md
-// your response here
+// find all patients
 ```
 
 We use the constant `Patient` in the `PatientsController`, but where is it
 actually defined?
 
 ```md
-// your response here
+// Patient model
 ```
 
 Why do our `model` files appear to be lacking so much code, when they're doing
 so much for us?
 
 ```md
-// your response here
+// they define the format of the code not the content
 ```
 
 What does typing `bin/rails dbconsole` in a directory that houses a rails app do for
 us?
 
 ```md
-// your response here
+// shows the outline of the database structure
 ```
 
 What does typing `rails db` in a directory that houses a rails app do for us?
 
 ```md
-// your response here
+// generate a blank db
```

Comment:
`rails db` opens up a PostgreSQL shell connected to the database that we're working with.